metals/src/main/scala/scala/meta/internal/metals/formatting/MultilineString.scala

```diff
@@ -276,6 +320,12 @@ case class MultilineString(userConfig: () => UserConfiguration)
     List(new TextEdit(new Range(pos1, pos2), "\"\"\"\"\"\""))
   }
 
+  private def addTripleQuote(pos: Position): List[TextEdit] = {
+    val pos1 = new Position(pos.getLine, pos.getCharacter)
+    val pos2 = new Position(pos.getLine, pos.getCharacter + 1)
+    List(new TextEdit(new Range(pos1, pos2), "\"\"\""))
```

Comment:
[nit]
```suggestion
    List(new TextEdit(new Range(pos, pos), "\"\"\""))
```

It's ok with point

Reply:
I am just afraid about removing the `pos.getCharacter + 1` for the end of range, because this was copied from existing logic and the indices in TextEdits are so tricky.